backend/controller/user.go

```diff
@@ -17,20 +17,31 @@ import (
 	"net/smtp"
 	"os"
 	"strconv"
+	"strings"
 	"time"
 )
 
-func GetAllUsers(w http.ResponseWriter, _ *http.Request) {
+func GetAllUsers(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	err, users := dbHelpers.GetAllTable(w, []models.User{})
-	if err != nil {
-		return
+	var users []models.User
+	// Get user ids from the query
+	userIdsParam := r.URL.Query().Get("users")
+	if userIdsParam == "" {
+		var err error
+		err, users = dbHelpers.GetAllTable(w, []models.User{})
+		if err != nil {
+			return
+		}
+	} else {
+		// Split the comma-separated string into individual IDs
+		userIds := strings.Split(userIdsParam, ",")
```

Comment:
does this give a proper error msg on string input?

Reply:
as in like incorrect input

Reply:
If the input is incorrect it returns no users

Reply:
probably more ideal to actually parse it: https://stackoverflow.com/questions/24972950/go-convert-strings-in-array-to-integer 
this seems helpful